lib/Core/SymbolicError.h

```diff
@@ -141,11 +141,11 @@ class SymbolicError {
   std::string &getOutputString() { return errorState->getOutputString(); }
 
   void executeStore(ref<Expr> address, ref<Expr> value, ref<Expr> error,
-                    ref<Expr> valueWithError);
+                    ref<Expr> valueWithError, KInstruction *i);
 
-  void storeError(ref<Expr> address, ref<Expr> error,
-                  ref<Expr> errorWithValue) {
-    errorState->executeStoreSimple(address, error, errorWithValue);
+  void storeError(ref<Expr> address, ref<Expr> error, ref<Expr> errorWithValue,
+                  KInstruction *i) {
```

Comment:
@Himeshi Here as well, `KInstruction *` can be changed with `llvm::Instruction`.